vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -36,16 +29,19 @@
     "numerical": (RangeSlider, Slider),
     "categorical": (Dropdown, Checklist, RadioItems),
     "temporal": (DatePicker,),
+    "boolean": (Switch,),
 }
 CategoricalSelectorType = Union[Dropdown, Checklist, RadioItems]
 NumericalTemporalSelectorType = Union[RangeSlider, Slider, DatePicker]
+BooleanSelectorType = Switch
 
 # This disallowed selectors for each column type map is based on the discussion at the following link:
 # See https://github.com/mckinsey/vizro/pull/319#discussion_r1524888171
 DISALLOWED_SELECTORS = {
     "numerical": SELECTORS["temporal"],
     "temporal": SELECTORS["numerical"],
     "categorical": SELECTORS["numerical"] + SELECTORS["temporal"],
+    "boolean": SELECTORS["temporal"] + SELECTORS["numerical"],
```

Comment:
```suggestion
    "boolean": SELECTORS["numerical"] + SELECTORS["temporal"],
```
Just to match the above line.